content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,671 @@
+```
+
+## Additional OTel components for Kubernetes
+
+This section covers additional OTel Collector components you can use to capture
+Kubernetes metrics.
+
+Receiving data:
+
+- [Kubernetes Cluster Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver):
+  collects cluster-level metrics and entity events from the
+  [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
+- [Kubernetes Objects Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver):
+  collects (pull/watch) objects from the Kubernetes API server
+- [Kubelet Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver):
+  pulls node, pod, container, and volume metrics from the API server on a
+  [Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
+  and sends it down the metric pipeline for further processing
+- [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver):
+  scrapes system metrics from hosts that make up the cluster
+
+Processing data:
+
+- [Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor):
+  adds Kubernetes context, thereby
+- enabling you to correlate application telemetry with your Kubernetes telemetry
+- – considered one of the most important components for monitoring Kubernetes
+- with OpenTelemetry
+
+You can also use the Kubernetes attributes processor to set custom resource
+attributes for traces, metrics, and logs using the Kubernetes labels and
+annotations you’ve added to your pods and namespaces.
+
+There are a few more Collector components you can implement to monitor
+Kubernetes, including Kubernetes-specific ones as well as general-use
+processors, such as the batch, memory limiter, and resource processors. To learn
+more, see
+[Important Components for Kubernetes](/docs/kubernetes/collector/components/).
+
+After you’ve configured the components in your Collector config file, you need
+to enable them within the [pipelines](/docs/collector/configuration/#pipelines)
+section. A data pipeline enables you to
+[collect](/docs/collector/configuration/#receivers),
+[process](/docs/collector/configuration/#processors), and route data from any
+source [to one destination or more](/docs/collector/configuration/#exporters).
+
+## Pros and cons
+
+The following are pros and cons of the setup we covered in this article.
+
+**Pros:**
+
+- Not having to maintain Prometheus as your data store, which means less
+  infrastructure overall to maintain -- especially if you go with an all-in-one
+  observability backend to ingest OTel data (traces, metrics, logs).
+- Not having to maintain the Prometheus Operator; while you would still have to
```

Comment:
You need to maintain the OTel operator, right?

Reply:
@avillela ^^

Reply:
Yes, totally true that you would still have to maintain the OTel Operator, but at least you're not having to maintain the Prometheus Operator AND the OTel Operator. I wonder if we just remove the part on maintaining the Prometheus Operator because it just adds confusion.